src/main/java/com/redhat/devtools/intellij/lsp4mp4ij/psi/internal/config/properties/MicroProfileConfigPropertyProvider.java

```diff
@@ -11,17 +11,7 @@
 
 import com.intellij.lang.jvm.JvmMember;
 import com.intellij.openapi.module.Module;
-import com.intellij.psi.JavaPsiFacade;
-import com.intellij.psi.PsiAnnotation;
-import com.intellij.psi.PsiClass;
-import com.intellij.psi.PsiElement;
-import com.intellij.psi.PsiField;
-import com.intellij.psi.PsiLocalVariable;
-import com.intellij.psi.PsiMember;
-import com.intellij.psi.PsiMethod;
-import com.intellij.psi.PsiModifierListOwner;
-import com.intellij.psi.PsiParameter;
-import com.intellij.psi.PsiVariable;
+import com.intellij.psi.*;
```

Comment:
I think your IJ replace with *. Is it a good pattern?

Reply:
To be consistent with the existing code and with LSP4MP, yes please.

Reply:
done